agent/tests/test_lib_aws_sdk_php.c

```diff
@@ -61,24 +61,83 @@ NR_PHP_WRAPPER(expect_arg_value_null) {
 }
 NR_PHP_WRAPPER_END
 
+NR_PHP_WRAPPER(aws_dynamodb_set_params_wrapper) {
+  // (void)wraprec;
```

Comment:
My editor complains about an unused `wraprec` parameter in the `NR_PHP_WRAPPER` call
```suggestion
  (void)wraprec;
```